src/Controller/SponsorshipBenefit/Delete.php

```diff
@@ -29,15 +30,15 @@ public function __construct(
         $this->csrfTokenManager = $csrfTokenManager;
     }
 
-    public function handle(Request $request, string $id): RedirectResponse
+    /**
+     * @param Request $request
+     * @param SponsorshipBenefit $sponsorshipBenefit
+     * @ParamConverter("sponsorshipBenefit", class="App:SponsorshipBenefit")
+     * @return RedirectResponse
+     */
+    public function handle(Request $request, SponsorshipBenefit $sponsorshipBenefit): RedirectResponse
```

Comment:
Maybe get the `$sponsorshipBenefit` from the manager (which is still available here) using `$request->query->get()` rather than using the paramconverter. Any thoughts @lbassin / @kevinjhappy ?
What we want to avoid is to pass an int as parameter, other solutions are ok.